language-plutus-core/src/Language/PlutusCore/Constant/Function.hs

```diff
@@ -77,31 +77,30 @@ typeSchemeResult (TypeSchemeArrow _ schB) = typeSchemeResult schB
 typeSchemeResult (TypeSchemeAllSize schK) = typeSchemeResult (schK ())
 
 -- | Convert a 'TypedBuiltin' to the corresponding 'Type'.
-typedBuiltinToType :: TypedBuiltin (Type TyName ()) a -> Quote (Type TyName ())
+typedBuiltinToType :: TypedBuiltin (Type TyName ()) a -> Type TyName ()
```

Comment:
Crossing language boundaries (compiling types that have a Haskell representation).